src/components/sidebar/sidebar.css

```diff
@@ -13,13 +13,14 @@
 
 .sidebar-calltree .sidebar-contents-wrapper {
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: min-content 1fr 1fr;
```

Comment:
I realize this is wrong now: I wrote this when I've put the category square first, but now it's at the end... As a result everything "jumps" sometimes.